package.json

```diff
@@ -66,6 +68,8 @@
     "kremling": "^2.0.1",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "single-spa-react": "^2.10.2"
+    "semver": "^6.3.0",
+    "single-spa-react": "^2.10.2",
+    "systemjs": "^6.1.2"
```

Comment:
This dependency is not imported anywhere. We do not need to add SystemJS as a dependency in the package.json file, since it's made globally available inside of openmrs-module-spa